app/invocation/cache_requests_card.tsx

```diff
@@ -304,6 +314,20 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
         <div>
           <DigestComponent hashWidth="96px" sizeWidth="72px" digest={result.digest} expandOnHover={false} />
         </div>
+        {this.isCompressedSizeColumnVisible() && (
```

Comment:
Are there any column headers? Might be nice as there are a lot more numbers now.

Reply:
Also trying to think through the order of the columns.

Seems like in order of importance it'd be something like:
Hit/Miss, Name, Digest/Size, Time, Compression savings, Waterfall

Reply:
Reordered the columns :+1: (tweaked the order slightly)